src/NzbDrone.Common.Test/Http/UserAgentParserFixture.cs

```diff
@@ -0,0 +1,22 @@
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Common.Http;
+using NzbDrone.Test.Common;
+
+namespace NzbDrone.Common.Test.Http
+{
+    [TestFixture]
+    public class UserAgentParserFixture : TestBase<UserAgentParser>
+    {
+        [TestCase("Mylar3/3ee23rh23irqfq/13123123/http://www.github.com/mylar3/mylar3/", "Mylar3")]
+        [TestCase("Lidarr", "Lidarr")]
```

Comment:
The arr ones have more info than this

Reply:
got those examples handy at all?

having issues trying to find any on my end and my prowlarr dev setup is not easily setup to test *arrs <-> prowlarr locally